py/picca/delta_extraction/data.py

```diff
@@ -600,3 +645,65 @@ def save_deltas(self):
             self.rejection_log.add_to_rejection_log(forest, "accepted")
 
         self.rejection_log.save_rejection_log()
+
+    def rename_exposures(self):
+        """In case there are not coadded forest, rename them to make independent the delta 
+        extraction
+        """
+        los_id_list = np.array([forest.los_id for forest in self.forests])
+        unique_los_id_list = np.unique(los_id_list)
+        if unique_los_id_list.size != los_id_list.size:
+            for los_id in unique_los_id_list:
+                for i, forest in enumerate(
+                    np.array(self.forests)[los_id_list == los_id]
+                ):
+                    forest.los_id = f"{los_id}_{i}"
```

Comment:
still not sure if this los_id hack for combining an exposure index with the los_id is a good thing to do (would e.g. actually prefer this to be two different properties and to not just give a random index i, but having the EXPID encoded in there). But should be good enough for the moment.